widgets/src/api/pda/processes.js

```diff
@@ -10,7 +10,6 @@ export const getProcesses = async connection =>
     uri: `/connections/${connection}/processes/definitions`,
     method: METHODS.GET,
     mockResponse: MOCK_PROCESSES,
-    useAuthentication: true,
```

Comment:
this was when I was testing, and that particular endpoint was not working because of useAuthentication. Maybe because of that CORS issue last week, but anyway I'll revert this

Reply:
reverted